tests/verbose_output.yml

```diff
@@ -0,0 +1,35 @@
+name: testsuite run in verbose mode
+testcases:
+- name: testsuite run in verbose mode
+  steps:
+  # spawn a venom sub-process and expect it to fail and make assertions on its error messages
+  # ensure no color to avoid annoying checks
+  - type: exec
+    script: NO_COLOR=1 {{.venom.executable}} run failing/verbose_output.yml {{.value.opt}}
```

Comment:
Can you add something on README about the NO_COLOR please?
In previous release of venom, we removed the color as it was not working on some environment (NO_COLOR did not exist at this time). We can re-add them, but need to provide a way to disable them, NO_COLOR is perfect for that.

Reply:
ok, I added it
Found it in https://github.com/fatih/color#disableenable-color for reference (the underlying lib used for logs)